src/cli.js

```diff
@@ -1,50 +1,67 @@
-#!/usr/bin/env node
-const fs = require('fs');
+const fs = require("fs");
 
 const translations = {
-    'mama aida hoilo': 'let', // Variable declaration
-    'bol toh mama': 'console.log', // Print to console
-    'kisuina mama': 'null', // Null value
-    'haw mama': 'true', // Boolean true value
-    'nah mama': 'false', // Boolean false value
-    'jodi mama': 'if', // If condition
-    'nah hoile mama': 'else if', // Else if condition
-    'akdom e nah hoile': 'else', // Else condition
-    'jotokhon porjonto mama': 'while', // While loop
-    'thamis mama': 'break', // Break statement
-    'tarpor er tah dekh': 'continue' // Continue statement
-    // ... other keywords as needed (Mama tui jodi aida aro kisu add korte chas mama, akta PR open kor mama!)
+  "mama aida hoilo": "let", // Variable declaration
+  "bol toh mama": "console.log", // Print to console
+  "kisuina mama": "null", // Null value
+  "haw mama": "true", // Boolean true value
+  "nah mama": "false", // Boolean false value
+  "jodi mama": "if", // If condition
+  "nah hoile mama": "else if", // Else if condition
+  "akdom e nah hoile": "else", // Else condition
+  "jotokhon porjonto mama": "while", // While loop
+  "thamis mama": "break", // Break statement
+  "tarpor er tah dekh": "continue", // Continue statement
+  "por por mama": "++", // Increment
+  "kome kome mama": "--", // Decrement
+  "thakbe mama": "function", // Function declaration
+  "kor mama": "", // Function call (You need to provide the function name and arguments)
+  "kore fel mama": "delete", // Delete variable
+  "koyta ache mama": ".length", // Length of an array or string
+  "shuru hoilo": "{", // Start of a block
```

Comment:
shuru hoilo, shesh hoilo (start and end of blocks) seems like a bit much? ai duita bad dei? baki gula shob legit kintu tor mama

Reply:
ho mama bad diya daw